cli/Cargo.toml

```diff
@@ -45,6 +45,9 @@ postgres = [
     "splinter/postgres",
 ]
 
+# Aliases - used to map libsplinter feature names to their CLI counterparts
+biome-credentials = ["database-migrate-biome-credentials"]
```

Comment:
This should be in a separate commit, since it relates only to features of the CLI

Reply:
Updated